### DISCUSSION TITLE
AX6编译出来5G和2.G是反着的，5G连接之后不能上网

### DISCUSSION THREAD

**Discussion by zjwxh** [Boos4721/openwrt#76] [Bugs/问题反馈]
AX6编译出来5G和2.G是反着的，5G连接之后不能上网，能修正下吗 ？

**Comment by Boos4721** [comment]
LOG

**Comment by zjwxh** [comment]
今天晚上编译AX6固件，5G网络连接后可以上网了

**Comment by idelame** [comment]
这个问题一直存在，编译后的名称是不对的。5G的写着openwrt_2.4G   真正2.4G的写的是5G

**Reply by Boos4721** [reply]
It isnt bug, bcz u need edit `radio0` to `radio1` for mac80211 config
